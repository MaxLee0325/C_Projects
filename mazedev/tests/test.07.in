Alpha
Bravo
Charlie
Delta
END
Alpha Bravo
Alpha Charlie
Alpha Delta
Bravo Alpha
Bravo Charlie
Bravo Delta
Charlie Alpha
Charlie Bravo
Charlie Delta
Delta Alpha
Delta Bravo
Delta Charlie
END
Alpha
Charlie
Delta
Bravo
Alpha
Bravo
END
