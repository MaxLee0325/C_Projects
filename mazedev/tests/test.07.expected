Entering room Alpha.
Entering room Charlie.
Entering room Delta.
Entering room Bravo.
Entering room Alpha.
Entering room Bravo.
Ended in room Bravo.
